test/typescript/axios.ts

```diff
@@ -251,6 +251,8 @@ instance1.post('/user', { foo: 'bar' }, { headers: { 'X-FOO': 'bar' } })
 
 // Defaults
 
+console.log(axios.defaults.headers['X-FOO']);
```

Comment:
Can you please remove the console log and the extra enter

Reply:
Done